allennlp/training/no_op_trainer.py

```diff
@@ -0,0 +1,38 @@
+import os
+from typing import Dict, Any
+
+from allennlp.common import Params
+from allennlp.models import Model
+from allennlp.training.checkpointer import Checkpointer
+from allennlp.training.trainer import TrainerPieces
+from allennlp.training.trainer_base import TrainerBase
+
+@TrainerBase.register("no_op")
+class NoOpTrainer(TrainerBase):
+    def __init__(self, serialization_dir: str, model: Model) -> None:
+        """
+        A trivial trainer to assist in making model archives for models that do not actually
+        require training. For instance, a majority class baseline.
+        """
+
+        super().__init__(serialization_dir, -1)
```

Comment:
might be worth using a keyword arg `cuda_device=-1`, otherwise the -1 is just a magical number

Reply:
Done!